vpplink/stats.go

```diff
@@ -38,3 +41,15 @@ func GetInterfaceStats(sc *statsclient.StatsClient) (ifNames adapter.NameStat, d
 	}
 	return ifNames, dumpStats, nil
 }
+
+func (v *VppLink) GetBufferStats() (uint32, uint32, uint32, error) {
+	response := &interfaces.GetBuffersStatsReply{}
```

Comment:
This function is missing the `v.lock... defer unlock` boilerplate (and there's also a missing newline at the end of the file).